### ISSUE TITLE
Development team

### ISSUE THREAD

**Issue by NikKovIos** [castus/Xcode8Extensions#2]
`Signing for "Xcode8Extensions" requires a development team. Select a development team in the project editor.`

What if i don't have a team?

**Comment by castus** [comment]
You always have a team. If you have personal Apple account, choose "Automatically manage signing" in General tab and select proper team (personal team).

**Comment by NikKovIos** [comment]
Yes, u're right. Had to set automating signing in project and target the both.
But the line in editor menu didn't appear(

**Comment by castus** [comment]
Hmm, I just opened it on Xcode 8.3.3, MacOS 10.12.5. In fact I must do it twice, before the command shows up, but it works as expected. Could you please double check on your side?
